packages/functional-tests/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint ."
   },
   "devDependencies": {
-    "@playwright/test": "^1.43.1",
+    "@playwright/test": "^1.50.1",
```

Comment:
We need yarn.lock changes to trigger an image build.

Reply:
I would do a smaller increase just in case, maybe 1.44